code/modules/materials/Mat_ProcsDefines.dm

```diff
@@ -352,6 +352,12 @@ var/global/list/material_cache = buildMaterialCache()
 		if(R.validate(M)) return R
 	return null
 
+/proc/findRecipeName(var/obj/item/One,var/obj/item/Two)
+	var/tempmerge = getFusedMaterial(One.material, Two.material)
+	for(var/datum/material_recipe/R in materialRecipes)
+		if(R.validate(tempmerge)) return R
```

Comment:
shouldn't this be `return R.getName()`?

Reply:
yeah probably.

Reply:
it doesnt work with R.getName() but it works the original way